.deepsource.toml

```diff
@@ -0,0 +1,15 @@
+version = 1
```

Comment:
Please remove this file from the PR, we don't plan on using your tool regularly and the repo has enough config files already :)

Reply:
Sure :)